variables.tf

```diff
@@ -53,6 +53,12 @@ variable "member_memory_mb" {
   }
 }
 
+variable "backup_crn" {
```

Comment:
a nice to have would be to add a regex check on the crn - eg: check it starts with "crn" and ends with ":backup:". That'll catch issues early in the flow.

Reply:
The backup doesn't end with `:backup:` although the documentation mentions it like that. 

It contains ":backup:" but doesn't end with it. The actual backup CRN of the PostgreSQL DB:

`crn:v1:bluemix:public:databases-for-postgresql:us-south:a/abac0df06b644a9cabc6e44f55b3880e:6a212da6-5b41-49f6-b8f2-881e1468d519:backup:cfd65a05-d6a7-4e84-a894-d8b662201f03
`

Reply:
If we really need one, I can have a check that it starts with `crn:` and _**contains**_ `:backup:`